src/rest/routes/deferred/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from "express";
+import { DeferredInstallationUrlRoute } from "./deferred-installation-url";
+import { DeferredRequestParseRoute } from "./deferred-request-parse";
+import { DeferredCheckOwnershipAndConnectRoute } from "./deferred-check-ownership-and-connect";
+import { JwtHandler } from "../../middleware/jwt/jwt-handler";
+import { GitHubTokenHandler } from "~/src/rest/middleware/jwt/github-token";
+
+export const DeferredRouter = Router({ mergeParams: true });
+
+DeferredRouter.get("/installation-url", JwtHandler, DeferredInstallationUrlRoute);
+
+DeferredRouter.get("/parse/:requestId", JwtHandler, DeferredRequestParseRoute);
```

Comment:
Should we remove JwtHandler?

Reply:
This has been removed in the [FE PR](https://github.com/atlassian/github-for-jira/pull/2508).

Reply:
I've just merged in the FE pr to this one, so this middleware has now been removed